### PR TITLE
TDL-6587 Modify 'invoice_line_items' stream on 'test_all_fields.py' to pass

### PR DESCRIPTION
**Pull request by prijendev**
# Description of change
- Two Fields named `tax_rates` and `price` have been added to the schema as it returned in the response from an API call using stripe’s python client and also mentioned on the stripe document. ([Link1](https://stripe.com/docs/api/invoices/line_item#invoice_line_item_object-price) , [Link2](https://stripe.com/docs/api/invoices/line_item#invoice_line_item_object-tax_rates))
- Removed the above fields from the KNOWN_MISSING_FIELDS map in test_all_fields.py.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/test_all_fields.py [hunk 1]
```diff
@@ -47,9 +47,7 @@
     },
     'plans': set(),
     'invoice_line_items': {
-        'tax_rates',
```
Comment (prijendev) [inline tests/test_all_fields.py:50]: `tax_rates` and `price` - both fields added to schema. Now it replicates in actual records also. So, removed it from the `KNOWN_MISSING_FIELDS` map.

File: tests/test_all_fields.py [hunk 2]
```diff
@@ -165,8 +162,7 @@
     'invoices': {'updated'},
     'plans': {'updated'},
     'invoice_line_items': {
-        'updated',
```
Comment (prijendev) [inline tests/test_all_fields.py:168]: Tap does write the `updated` field in `invoice_line_items` as it has no replication key value in the object itself available. `invoice_line_items` is bookmarked based on parent invoices. [Reference](https://github.com/singer-io/tap-stripe/blob/master/tap_stripe/__init__.py#L62)

File: tests/test_all_fields.py [hunk 3]
```diff
@@ -406,10 +408,14 @@ def all_fields_test(self, streams_to_test):
                         #      we will log the inequality and assert that the datatypes at least match.
 
                         for field in set(actual_record.keys()).difference(field_adjustment_set):  # skip known bugs
+
                             with self.subTest(field=field):
                                 base_err_msg = f"Stream[{stream}] Record[{pks_tuple}] Field[{field}]"
 
                                 expected_field_value = expected_record.get(field, "EXPECTED IS MISSING FIELD")
+                                if field == "price" and expected_field_value != "EXPECTED IS MISSING FIELD" and expected_field_value.get('created'):
```
Comment (karanpanchal-crest) [inline tests/test_all_fields.py:416]: @prijendev Why do we check `field == "price"` in this case?

**Review by karanpanchal-crest — CHANGES_REQUESTED** [review]

File: tests/test_all_fields.py (hunk 3 above)
Comment (prijendev) [reply tests/test_all_fields.py:416]: Here, it compares the field value of records retrieved through API(expected records) and records written in the target(during sync mode - actual records). `price` field is an object and contains a nested field `created`. API returns the value of `created` in timestamp while tap transforms it into the date-time format("%Y-%m-%dT%H:%M:%S.000000Z"). That's, why here we are converting the value of the `price. created` field returned by API into the date-time format.

Comment (karanpanchal-crest) [reply tests/test_all_fields.py:416]: Add a code comment that this check is specific for "invoice_line_items" stream for better understanding

**Review by karanpanchal-crest — CHANGES_REQUESTED** [review]

File: tests/test_all_fields.py [hunk 4]
```diff
@@ -247,6 +243,12 @@ def getPKsToRecordsDict(self, stream, records, duplicates=False):  # BUG_9720
         return pks_to_record_dict_1, pks_to_record_dict_2
 
 
+    def epoch_to_dt(self, epoch_ts):
```
Comment (dbshah1212) [inline tests/test_all_fields.py:246]: We have one "time converter" in base.py as well, Can we use that?

File: tests/test_all_fields.py (hunk 3 above)
Comment (prijendev) [reply tests/test_all_fields.py]: Removed these changes and moved the `price` field in `records_data_type_conversions` to convert timestamp value to date-time format. Added comment in the code. [Reference](https://github.com/singer-io/tap-stripe/blob/master/tests/base.py#L380)

File: tests/test_all_fields.py (hunk 4 above)
Comment (prijendev) [reply tests/test_all_fields.py]: Yes, updated it.

**Review by karanpanchal-crest — APPROVED** [review]

**Review by kspeer825 — APPROVED** [review]

**Review by KrisPersonal — APPROVED** [review]

**Pull request merged**